turbo.json

```diff
@@ -7,7 +7,8 @@
     "tsconfig.json",
     "scripts/patch-manifests.js",
     "scripts/banner.js",
-    "patches/*"
+    "patches/*",
+    "packages/services/api/src/shared/*"
```

Comment:
Why? I guess what you tried to fix is caused by having relative imports, we should stick to `import { ... } from '@hive/xyz'` instead of `import { ... } from '.././packages/xyz'` and we will follow what Turbo expects from us.